assembl/static2/js/app/components/common/wordCloud/wordCloud.scss

```diff
@@ -0,0 +1,7 @@
+@import url(//fonts.googleapis.com/css?family=Lato);
```

Comment:
Please remove this import. We already load Lato fonts via `#{$fonts-dir}` and some clients doesn't want external resources.